<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tables</class>
 <widget class="QMainWindow" name="Tables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget_display_tables">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="l_dialogTitle">
      <property name="text">
       <string>Tabellen ansehen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_table">
      <property name="placeholderText">
       <string>Tabellen...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tv_table"/>
    </item>
    <item>
     <widget class="QPushButton" name="pb_download">
      <property name="text">
       <string>Ausgewähltes herunterladen</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_editMode">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Bearbeitung aktivieren</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Änderungen speichern</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cb_table</tabstop>
  <tabstop>tv_table</tabstop>
  <tabstop>pb_download</tabstop>
  <tabstop>pb_save</tabstop>
  <tabstop>cb_editMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
